### DISCUSSION TITLE
Icon request: icon-mouse-scroll-down

### DISCUSSION THREAD

**Discussion by radubr** [FortAwesome/Font-Awesome#21159] [Icon Requests]
This icon can be used on Bootstrap Jumbotron heading to indicate to the user that more content is available bellow and he has to scroll down to see it.

 ![Image of scroll down icon by Stephan Wetz - https://dribbble.com/trashpixels](https://d13yacurqjgara.cloudfront.net/users/4178/screenshots/1264403/scroll_down_ddribbble.png)

**Comment by tagliala** [comment]
The request is valid but I have concerns on the use case.

...and if there is not a mouse (touch devices or touchpad)?

I "solved" this issue by using `fa-long-arrow-down` and a custom animation, example: http://jsfiddle.net/tagliala/77re29ct/40/

**Comment by radubr** [comment]
I think that just an arrow with 3 bars on top, without the mouse, to indicate to scroll down will suffice for all devices (desktops, laptops, mobiles, tablets, TVs).

**Comment by hhtrs** [comment]
how can I download?

**Comment by chnwentao** [comment]
+1

**Comment by adamdelaney** [comment]
+1

**Comment by adaniello** [comment]
+1

**Comment by goowikns** [comment]
+1

**Comment by razor-rich** [comment]
+1

**Comment by mdeboer** [comment]
+1

**Comment by dextermb** [comment]
+1

**Comment by PTiCA1** [comment]
My solution using CSS3
![stack Overflow](https://github.com/PTiCA1/Mouse-Animation-Scroll-CSS/blob/master/mouse-animation-scroll.gif?raw=true)

[Mouse Animation Scroll](https://github.com/PTiCA1/Mouse-Animation-Scroll-CSS)
live preview on [Codepen.io](http://codepen.io/PTiCA1/pen/vKxVME)

**Comment by k0va1** [comment]
+1

**Comment by pansotdev** [comment]
+1

**Comment by adaniello** [comment]
+1

**Comment by Sreelakshmigc** [comment]
+1

**Comment by aethior** [comment]
Related (not duplicate):
- https://github.com/FortAwesome/Font-Awesome/issues/5308
- https://github.com/FortAwesome/Font-Awesome/issues/13250

@tagliala `devices-hardware (category)` label for this request?

**Comment by NielsHolt** [comment]
Hi
The center button is in v6 as computer-mouse-scrollwheel
I would suggest to name the two new icons
computer-mouse-left 
computer-mouse-right

@tagliala - I am not sure what label to add

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
